packages/jsii-pacmak/test/expected.jsii-calc-base/python/setup.py

```diff
@@ -32,7 +32,7 @@
     "install_requires": [
         "jsii~=0.20.8",
         "publication>=0.0.3",
-        "scope.jsii-calc-base-of-base~=0.20.8"
+        "scope.jsii-calc-base-of-base==0.20.8"
```

Comment:
The original dependency is a caret dependency (`^`), does `==` means the same thing in Python?

https://github.com/aws/jsii/blob/8fc81ca1b5623ca5a8d51a0aa62a125b0681ef94/packages/jsii-calc-base/package.json#L32

Reply:
Yeah that looks suspicious; but I reckon because the `jsii` compiler forwards the "resolved" (not "declared") version. That needs to be fixed as well (possibly in a separate PR, but then we probably shouldn't merge this without that).